pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -375,6 +375,10 @@ type TidbClusterSpec struct {
 
 	// PreferIPv6 indicates whether to prefer IPv6 addresses for all components.
 	PreferIPv6 bool `json:"preferIPv6,omitempty"`
+
+	// WaitForDnsNameIpMatchOnStartup indicates whether PD and TiKV has to wait
+	// until local IP address matches the one published to external DNS
+	WaitForDnsNameIpMatchOnStartup bool `json:"waitForDnsNameIpMatchOnStartup,omitempty"`
```

Comment:
How about adding a comment to indicate it is only supported in start script v2 now?

Reply:
done

Reply:
I didn't see any new commit after this comment.

Reply:
> I didn't see any new commit after this comment.

sorry, push did not go through first time.
now changes are there

Reply:
you need to re-generate OpenAPI spec.

Reply:
what do you think if I change `WaitForDnsNameIpMatchOnStartup bool` with more generic field `StartupScriptFeatureFlags []string `json:"startupScriptFeatureFlags, omitempty"`` ? This will allow us to enable features without extra fields to CRD in future.